_data/changelogs/docs-showcase.yml

```
@@ -2,6 +2,11 @@ title: Showcase
 type: documentation
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added Export Compliance Assistance Program (ECAP) to the showcase page.
+    affectsGuidance: true
+    githubPr: 2839
+    githubRepo: uswds-site
```

```suggestion
  - date: 2025-01-31
    summary: Added Export Compliance Assistance Program (ECAP) to the showcase page.
    githubPr: 2839
    githubRepo: uswds-site
```

Remove `guidance` tag and update date